app/src/main/res/values-en/strings.xml

```diff
@@ -113,30 +113,30 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="at_housename">house name %s</string>
     <string name="at_housenumber">house number %s</string>
     <!-- when the user left the answer form empty -->
-    <string name="no_changes">"You did not give any answer"</string>
+    <string name="no_changes">"You did not give an answer"</string>
 
     <string name="close">"close"</string>
     <string name="confirmation_discard_title">"Discard input?"</string>
     <string name="confirmation_discard_positive">"Discard"</string>
     <string name="short_no_answer_on_button">No</string>
 
     <string name="quest_leave_new_note_title">"Leave a note instead?"</string>
-    <string name="quest_leave_new_note_description">"You can leave a public note at this location which other mappers can comment on and hopefully resolve or alternatively just hide the quest for yourself only"</string>
-    <string name="quest_leave_new_note_yes">"OK"</string>
-    <string name="quest_leave_new_note_no">"No, just hide"</string>
+    <string name="quest_leave_new_note_description">"You can leave a public note for other mappers to resolve at this location, or hide this quest"</string>
+    <string name="quest_leave_new_note_yes">"Leave note"</string>
+    <string name="quest_leave_new_note_no">"Hide this quest"</string>
```

Comment:
I think the fact that the quest is only hidden for the current user ("yourself") should be mentioned.

Reply:
That's specifically why I preferred "stop seeing this quest" which clearly implies that it's for you only.  Perhaps use that language and "Stop seeing this quest" as the no option?

Reply:
Too long, this is displayed on a button

Reply:
Then my recommendation would be:
> You can leave a public note for other mappers to resolve at this location, or stop seeing this quest

> Leave note

> Hide

Reply:
"Hide" is not mentioned in the description, but I guess it will work anyway.